scripts/Readme.md

```diff
@@ -0,0 +1,131 @@
+# FHIR-Bulk Loader Getting startd scripts Readme
+Script purpose, order of execution and other steps necessary to get up and running with FHIR-Loader
+
+## Errata 
+There are no open issues at this time. 
+
+## Prerequisites 
+
+These scripts will gather (and export) information necessary to the proper deployment and configuration of Azure Healthcare API for FHIR, an Application Service Client, Key Vault and Resource Groups secure information will be stored in the Keyvault.  
+ - Prerequisites:  User must have rights to deploy resources at the Subscription scope 
+
+__Note__
+A Keyvault is necessary for securing Service Client Credentials used with the FHIR Service and FHIR-Proxy.  Only 1 Keyvault should be used as this script scans the keyvault for FHIR Service and FHIR-Proxy values. If multiple Keyvaults have been used, please use the [backup and restore](https://docs.microsoft.com/en-us/azure/key-vault/general/backup?tabs=azure-cli) option to copy values to 1 keyvault.
+
+__Note__ 
+The FHIR-Bulk Loader & Export scripts are designed for and tested from the Azure Cloud Shell - Bash Shell environment.
+
+
+### Naming & Tagging
+All Azure resource types have a scope that defines the level that resource names must be unique.  Some resource names, such as PaaS services with public endpoints have global scopes so they must be unique across the entire Azure platform.    Our deployment scripts strive to suggest naming standards that group logial connections while aligning with Azure Best Practices.  Customers are prompted to accept a default or suppoly their own names during installation, examples include:
+
+Prefix      | Workload        |  Number     | Resource Type 
+------------|-----------------|-------------|---------------
+NA          | fhir            | random      | NA 
+User input  | secure function | random      | storage 
+
+Resources are tagged with their deployment script and origin.  Customers are able to add Tags after installation, examples include::
+
+Origin              |  Deployment       
+--------------------|-----------------
+HealthArchitectures | FHIR-Bulk   
+
+---
+
+## Setup 
+Please note you should deploy these components into a tenant and subscriotion where you have appropriate permissions to create and manage Application Registrations (ie Application Adminitrator RBAC Role), and can deploy Resources at the Subscription Scope. 
```

Comment:
Typos:
* **and subscriotion** should be subscription
* **ie Application Adminitrator** should be Administrator

Reply:
corrected subscription and an additional typo.  Last note either Application Admin or Global Admin works.  I made the correction